tests/test_market.py

```diff
@@ -168,32 +168,29 @@ def test_market_offer(market, offer):
 
 
 def test_market_bid(market: TwoSidedPayAsBid):
-    bid = market.bid(1, 2, 'bidder', 'seller', 'bidder')
+    bid = market.bid(1, 2, 'bidder', 'bidder')
```

Comment:
I wonder why is 'bidder' repeated 2 times here?

Reply:
If you look at the two sided market function `bid` you see that there are 4 positional arguments:
`price: float, energy: float, buyer: str, buyer_origin` hence, the last `bidder` is the buyer_origin.
It is ok that these are equal as this is the initial bit (which was not forwarded yet).